crates/sdk/src/queries/shell/eth_bridge.rs

```diff
@@ -254,12 +254,10 @@ where
     };
 
     // check which transfers in the Bridge pool match the requested hashes
+    let last_committed_height = ctx.state.in_mem().get_last_block_height();
```

Comment:
I believe the behavior that users want is to get back the height that heir query is targeting. So I think it's ok to return the last block height if the query contains a height of `0` but for all the other cases we should set the response height to be the same as the query one

Reply:
ah, you're right. I fixed them. https://github.com/anoma/namada/pull/3164/commits/801a115c388b2b9b514702914ba5db965996644a